lib/prepare/loadConfig.js

```diff
@@ -40,13 +40,20 @@ function parseConfig (file) {
     data = tomlParser.parse(content)
     // reformat to match config since TOML does not allow different data type
     // https://github.com/toml-lang/toml#array
-    const format = []
-    Object.keys(data.head).forEach(meta => {
-      data.head[meta].forEach(values => {
-        format.push([meta, values])
+    if (data.head) {
+      const format = []
+      Object.keys(data.head).forEach(meta => {
+        const item = data.head[meta]
+        if (Array.isArray(item)) {
+          item.forEach(values => {
+            format.push([meta, values])
+          })
+        } else {
+          format.push([meta, item])
```

Comment:
Why do you carry other things in a bug-fixed PR?

Reply:
Please remove it and open another feature request

Reply:
done.